src/equations/compressible_euler_multicomponent_2d.jl

```diff
@@ -1,67 +1,102 @@
 
 @doc raw"""
-    CompressibleEulerMulticomponentEquations2D
+    CompressibleEulerMulticomponentEquations2D(; gammas, gas_constants)
 
 !!! warning "Experimental code"
     This system of equations is experimental and can change any time.
 
-Multicomponent version of the compressible Euler equations for an ideal gas in two space dimensions.
+Multicomponent version of the compressible Euler equations 
+```math
+\begin{equation*}
+\partial t
+\begin{pmatrix}
+\rho v_1 \\ \rho v_2 \\ E \\ \rho_1 \\ \rho_2 \\ \vdots \\ \rho_{n}
+\end{pmatrix}
++
+\partial x
+\begin{pmatrix}
+\rho v_1^2 + p \\ \rho v_1 v_2 \\ (E+p) v_1 \\ \rho_1 v_1 \\ \rho_2 v_1 \\ \vdots \\ \rho_{n} v_1
+\end{pmatrix}
++
+\partial y
+\begin{pmatrix}
+\rho v_1 v_2 \\ \rho v_2^2 + p \\ (E+p) v_2 \\ \rho_1 v_2 \\ \rho_2 v_2 \\ \vdots \\ \rho_{n} v_2
+\end{pmatrix}
+= 
+\begin{pmatrix}
+0 \\ 0 \\ 0 \\ 0 \\ 0 \\ \vdots \\ 0
+\end{pmatrix}
+\end{equation*}
+```
+for calorically perfect gas in two space dimensions.
+
+The specific heat ratios 'gammas' and the gas constants 'gas_constants' in [kJ/(kg*K)]
+should be passed as tuples e.g. gammas=(1.4, 1.667).
```

Comment:
```suggestion
In case of more than one component, the specific heat ratios `gammas` and the gas constants
`gas_constants` (in [kJ/(kg*K)]) should be passed as tuples, e.g., `gammas=(1.4, 1.667)`.
```

Reply:
done in c7e9a75